code.gs

```diff
@@ -1659,6 +1659,38 @@ var i18n = {
     'It looks like you are using an outdated version of this script': 'Parece que tens uma versão desatualizada deste script',
     'You can find the latest one here': 'Podes encontrar a última versão aqui',
   },
+  'th': {
+    'Age': 'อายุ',
+    'Years': 'ปี',
+    'Events': 'อีเวนท์',
+    'Birthdays today': 'วันเกิดวันนี้',
+    'Birthdays tomorrow': 'วันเกิดพรุ่งนี้',
+    'Birthdays in {0} days': 'วันเกิดในอีก {0} วัน',
+    'Anniversaries today': 'วันครบรอบวันนี้',
+    'Anniversaries tomorrow': 'วันครบรอบพรุ่งนี้',
+    'Anniversaries in {0} days': 'วันครบรอบในอีก {0} วัน',
+    'Custom events today': 'อีเวนท์ที่กำหนดเองวันนี้',
+    'Custom events tomorrow': 'อีเวนท์ที่กำหนดเองวันพรุ่งนี้',
+    'Custom events in {0} days': 'อีเวนท์ที่กำหนดเองในอีก {0} วัน',
+    'Hey! Don\'t forget these events': 'เฮ้! อย่าลืมอีเวน์เหล่านี้ล่ะ',
+    'version': 'เวอร์ชั่น',
+    'dd-MM-yyyy': 'วว-ดด-ปปปป',
+    'Mobile phone': 'เบอร์โทรศัพท์',
+    'Work phone': 'เบอร์โทรศัพท์ที่ทำงาน',
+    'Home phone': 'เบอร์โทรศัพท์บ้าน',
+    'Main phone': 'เบอร์โทรศัพท์หลัก',
+    'Other phone': 'เบอร์โทรศัพท์อื่นๆ',
+    'Home fax': 'แฟกซ์บ้าน',
+    'Work fax': 'แฟกซ์ที่ทำงาน',
+    'Google voice': 'Google voice',
+    'Pager': 'เพจเจอร์',
+    'Home email': 'อีเมลบ้าน',
+    'Work email': 'อีเมลที่ทำงาน',
+    'Other email': 'อีเมลอื่นๆ',
+    'It looks like you are using an outdated version of this script': 'ดูเหมือนว่าคุณกำลังใช้เวอร์ชั่นเก่าสำหรับสคริปท์นี้',
+    'You can find the latest one here': 'คุณสามารถหาเวอร์ชั่นใหม่ได้ที่นี่',
+  },
+  
```

Comment:
Could you please remove this empty line?

Reply:
Got it :)